airbyte-integrations/connectors/source-scryfall/README.md

```diff
@@ -0,0 +1,33 @@
+# Scryfall
+This directory contains the manifest-only connector for `source-scryfall`.
+
+For Magic The Gathering fans. Here is a simple data source for all the cards and sets!
```

Comment:
Haha, good one! You still play, @michel-tricot?

Reply:
Not really, used to 20y ago and introduced my son to it a few months ago :)